.github/pull_request_template.md

```diff
@@ -0,0 +1,14 @@
+Pact Pull Request
+---
+
+Please include the following checklist in your PR:
+
+* [ ] Test coverage for the proposed changes
```

Comment:
For your consideration

```suggestion
* [ ] Test coverage for the proposed changes
* [ ] PR description contains example output from repl interaction or a snippet from unit test output
```

Reply:
I like this change, making it easier to verify the correct behaviour and understanding of the intended changes.